### ISSUE TITLE
[HL2] Entanglement: Alyx has purple eyes?

### ISSUE THREAD

**Issue by ComputerDruid** [ValveSoftware/Source-1-Games#863]
```
] getpos
setpos 1052.057617 248.031250 -639.968750;setang 6.448867 13.314090 0.000000
] status
hostname: Half-Life 2
version : 1745010/24 5317 insecure
# userid name                uniqueid            connected ping loss state  adr
map     : d2_prison_06 at: 1052 x, 248 y, -639 z
#     10 "The Computer Druid" STEAM_ID_PENDING   01:47       15    0 active loopback
players : 1 (1 max)
```

![2013-06-01_00002](https://f.cloud.github.com/assets/34696/595719/ef403e92-cb22-11e2-841c-9b6af3a7e1be.jpg)

![2013-06-01_00004](https://f.cloud.github.com/assets/34696/595727/0d60a16e-cb23-11e2-901c-cf3380bcf0d8.jpg)

System Information:

```
Processor Information:
    Vendor:  AuthenticAMD
    Speed: 2800 Mhz
    6 logical processors
    6 physical processors
    HyperThreading:  Unsupported
    FCMOV:  Supported
    SSE2:  Supported
    SSE3:  Supported
    SSSE3:  Unsupported
    SSE4a:  Supported
    SSE41:  Unsupported
    SSE42:  Unsupported

Network Information:
    Network Speed:  

Operating System Version:
    "NAME=Gentoo" (64 bit)
    Kernel Name:  Linux
    Kernel Version:  3.9.4
    X Server Vendor:  The X.Org Foundation
    X Server Release:  11401000
    X Window Manager:  KWin
    Steam Runtime Version:  steam-runtime-release-i386_2013-05-08

Video Card:
    Driver:  NVIDIA Corporation GeForce GTS 450/PCIe/SSE2

    Driver Version:  4.3.0 NVIDIA 319.23
    Desktop Color Depth: 24 bits per pixel
    Monitor Refresh Rate: 59 Hz
    VendorID:  0x10de
    DeviceID:  0xdc4
    Number of Monitors:  1
    Number of Logical Video Cards:  1
    Primary Display Resolution:  1920 x 1080
    Desktop Resolution: 1920 x 1080
    Primary Display Size: 20.08" x 11.42"  (23.07" diag)
                                            51.0cm x 29.0cm  (58.6cm diag)
    Primary Bus: PCI Express 16x
    Primary VRAM: 1024 MB
    Supported MSAA Modes:  2x 4x 8x 16x 

Sound card:
    Audio device: Realtek ALC892

Memory:
    RAM:  7987 Mb

Miscellaneous:
    UI Language:  English
    LANG:  en_US.utf8
    Microphone:  Not set
    Total Hard Disk Space Available:  30109 Mb
    Largest Free Hard Disk Block:  749 Mb

Installed software:

Recent Failure Reports:
    Wed May 29 19:18:52 2013 GMT: file ''/tmp/dumps/crash_20130529151849_1.dmp'', upload yes: ''CrashID=bp-41ef2fcd-976a-4fc9-91b4-1d5de2130529''
    Wed May 29 19:19:29 2013 GMT: file ''/tmp/dumps/crash_20130529151927_1.dmp'', upload yes: ''CrashID=bp-d686921c-b0c7-4e02-8e46-7af1d2130529''
    Fri May 31 02:35:21 2013 GMT: file ''/tmp/dumps/crash_20130530223521_1.dmp'', upload no: ''HTTP response code said error''
    Sun Jun  2 00:40:03 2013 GMT: file ''/tmp/dumps/crash_20130601204002_1.dmp'', upload no: ''HTTP response code said error''
```

**Comment by kisak-valve** [comment]
Closing pending steps to reproduce this issue.

**Comment by dgladkov** [comment]
I also experienced the problem, reloading my save didn't resolve the issue, however restarting the game fixed it for me. I tried reloading saves from previous maps and playing up until `d2_prison_06`, but I couldn't reproduce it either, so it appears that the issue was caused by extended play.

I got the issue by playing from `d2_prison_03` up to first encounter with Alyx in `d2_prison_06` in one session. Will try to reproduce it by playing the same segment again.
